CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0] - 2024-10-16
```

Comment:
```suggestion
```

Remove as release automation will add this when we release.

Reply:
All changes to the file have been reverted